episodes/access_hub.md

```diff
@@ -0,0 +1,204 @@
+
+:::{note}
+Your GitHub Team is defined and specially linked to your hub when the hub was initally deployed. This information should be known to Hub Champions. If not, please contact your community representative who deployed the hub.
+:::
+
+Follow the guidance below to practise granting user access to the [Community Showcase Hub](https://showcase.2i2c.cloud/), specifically
+
+- adding (but *not* removing for now) users to a GitHub Organization
+- adding and removing users to and from a GitHub Team.
+
+:::{danger}
+Please refrain from removing other users from the GitHub Organization during the training! The steps in {ref}`Removing members from the organization <org-remove>` are for completeness when it comes to administering your own hub outside of the training.
+
+:::
+
+## Managing your GitHub Organization
+
+- View a list of all of the GitHub Organizations that you are a member of by visiting [https://github.com/settings/organizations](https://github.com/settings/organizations).
+
+:::{figure} ../media/episodes/administer_hub/gh-org-1.png
+:width: 100%
+:alt: Screenshot showing a list of GitHub Organizations on https://github.com/settings/organizations.
+
+Screenshot showing an example list of GitHub Organizations on [https://github.com/settings/organizations](https://github.com/settings/organizations).
+:::
+
+- Click on the link to the [2i2c-community-showcase](https://github.com/2i2c-community-showcase) GitHub Organization (if you cannot see this link then please contact your instructor for access).
+
+- Click on the *People* menu tab to view a list of current Organization members.
+
+:::{figure} ../media/episodes/administer_hub/gh-org-2.png
+:name: gh-org-people
+:width: 100%
+:alt: Screenshot showing a list of members of the 2i2c-community-showcase GitHub Organization.
+
+Screenshot showing a list of [2i2c-community-showcase](https://github.com/orgs/2i2c-community-showcase/people) GitHub Organization members. The *People* tab menu is located at the top; the green *Invite member* button is located in the top right; the *Failed invitations* tab is located in the left sidebar and the *Member settings* ![member settings button](../media/episodes/administer_hub/gh-org-3.png) icon is located next to each member account.
+
+:::
+
+::::{dropdown} Exercise: Invite non-members to the organization
+:open:
+
+Check that you have *Owner* permissions for this organization to invite non-members to it. If not, contact an owner of the organization who can give you *Owner* permissions.
+
+- Invite a non-member to the organization by clicking the green *Invite member* button and searching by username, full name or email address.
+
+  :::{tip}
+  We recommend searching for and adding the `jnywong-test` username for the purposes of this training.
```

Comment:
Will this work? What happens if somebody tries to invite an account that already has an invitation pending?

Reply:
You're right, I've just tested out this workflow again and inviting an account with a pending invitation will show a slightly different screen that allows an Owner to edit and update the invitation. 

I'll add this info to the tip – I think "simulating" this experience on a dummy account is still useful.

Reply:
Tip information updated in https://github.com/czi-catalystproject/hub-champion-training/pull/40/commits/212b34ff4189382c0279bf401b851e2359e84048